src/components/grid/grid.stories.mdx

```diff
@@ -408,72 +397,68 @@ This example is best viewed in the Canvas tab using full-screen mode with device
   </Story>
 </Preview>
 
-## GridContainer Props
+## Props
 
+### GridContainer 
 <Props of={GridContainer} />
 
-## GridItem Props
-
+### GridItem 
 <Props of={GridItem} />
 
-### responsiveSettings Props
-
-<ArgsTable
-  rows={[
-    {
-      name: "alignSelf",
-      type: { summary: "string" },
-      description:
-        "How the grid item is aligned along the block (column) axis. Values: start, end, center, stretch",
-    },
-    {
-      name: "gridColumn",
-      type: { summary: "string" },
-      description:
-        'Starting and ending column position of the GridItem within the GridContainer separated by "/"',
-    },
-    {
-      name: "gridRow",
-      type: { summary: "number" },
-      description:
-        'Starting and ending row position of the GridItem within the GridContainer separated by "/"',
-    },
-    {
-      name: "justifySelf",
-      type: { summary: "string" },
-      description:
-        "How the grid item is aligned along the inline (row) axis. Values: start, end, center, stretch",
-    },
-    {
-      name: "maxWidth",
-      type: { summary: "string" },
-      description:
-        "Maximum width of the screen to which these rules to be applied",
-    },
-    {
-      name: "p",
-      type: { summary: "string | number" },
-      description: "Any valid CSS value to override default padding",
-    },
-    {
-      name: "pl",
-      type: { summary: "string | number" },
-      description: "Any valid CSS value to override default padding-left",
-    },
-    {
-      name: "pr",
-      type: { summary: "string | number" },
-      description: "Any valid CSS value to override default padding-right",
-    },
-    {
-      name: "pt",
-      type: { summary: "string | number" },
-      description: "Any valid CSS value to override default padding-top",
-    },
-    {
-      name: "pb",
-      type: { summary: "string | number" },
-      description: "Any valid CSS value to override default padding-bottom",
-    },
-  ]}
+### responsiveSettings
+<PropsTable
```

Comment:
This PropsTable isn't showing in when Storybook renders this MDX. 
![Screenshot 2021-03-05 at 14 27 19](https://user-images.githubusercontent.com/35449967/110128654-f570ca00-7dbe-11eb-857d-10695d75afbb.png)